lib/fpm/package/deb.rb

```diff
@@ -710,6 +712,18 @@ def fix_dependency(dep)
     end
   end # def fix_dependency
 
+  def validate_dependencies()
+    self.dependencies.each do |dep|
+      if (dep !~ /^[A-Z]/ && dep !~ /^[a-z]/)
+        logger.warn("Dependency '#{dep}' is not a valid deb dependency " \
+              "name and will be removed.")
+        self.dependencies.delete(dep)
+      end
+    end.flatten
+    # Remove rpmlib dependencies leftover from the rpm.
+    self.dependencies.delete_if { |d| d.start_with?('rpmlib') }
```

Comment:
This might benefit from being implemented in [converted_from](https://github.com/jordansissel/fpm/blob/master/lib/fpm/package/deb.rb#L234-L239) since it's specific to rpm->deb conversion.

Reply:
See comment below.

Reply:
See comment below.